core-web/libs/sdk/react/src/lib/next/hooks/useCheckHaveContent.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState, RefObject } from 'react';
+
+export const useCheckVisibleContent = (ref: RefObject<HTMLDivElement>) => {
+    const [haveContent, setHaveContent] = useState<boolean>(false);
+
+    useEffect(() => {
+        if (!ref.current) {
+            return;
+        }
+
+        const childElement = ref.current.firstElementChild;
```

Comment:
Sometimes for this kind of DOM operations is recomended to use `useLayoutEffect` which triggers after the DOM rendered

Reply:
Done!